src/guide/build-and-install.md

```diff
@@ -178,6 +179,27 @@ then the issue runs deeper.
 
 :::
 
+## Install OpenSSL
+
+Make sure you have OpenSSL installed. Note that in most Linux setups it is
+already available to you.
+
+- Instal OpenSSL on Ubuntu:
+
+  ```bash
+  sudo apt-get install libssl-dev
+  ```
+
+- Instal OpenSSL on macOS:
+
+  ```bash
+  brew install openssl
```

Comment:
I'm not sure that the user will have brew installed, because we haven't mentioned it before.

Reply:
linked to to the docs where users can find the install command and other info